src/main/webapp/help-testProjectUrl.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Specify path (URL) to the jar with your test project
+    Specify path (URL) to the jar with your test project. (URL must begin with 'file://')
```

Comment:
If I'm not mistaken, current implementation works only with local files. If http url will be passed, it must be downloaded firstly. Am I right? Or it already works with http urls also?